pkg/descheduler/strategies/duplicates.go

```diff
@@ -36,6 +36,8 @@ import (
 	"sigs.k8s.io/descheduler/pkg/utils"
 )
 
+const RemoveDuplicatesStrategyName = "RemoveDuplicatePods"
```

Comment:
Would you please remove all the `const *StrategyName` lines before the final squashing?